draft-ietf-tls-tls13.md

```diff
@@ -4215,30 +4215,28 @@ user_canceled
   SHOULD be followed by a "close_notify". This alert is generally a warning.
 {:br }
 
-Either party MAY initiate a close by sending a "close_notify" alert. Any data
-received after a closure alert MUST be ignored. If a transport-level close is
-received prior to a "close_notify", the receiver cannot know that all the
-data that was sent has been received.
+Either party MAY initiate a close of its write side of the connection by
+sending a "close_notify" alert. Any data received after a closure alert MUST
+be ignored. If a transport-level close is received prior to a "close_notify",
```

Comment:
I would change this to say "after a closure alert has been received" just to be totally clear.

Reply:
Agreed, fixed.